doc/run-mock-sds-endpoint.md

```diff
@@ -0,0 +1,23 @@
+# Running the Mock SDS endpoint
+
+In order to test loading supplementary data, we have a development script that creates a Mock SDS endpoint in Flask that
+returns mocked supplementary data.
+
+Ensure the following env var is set before running the script:
+```bash
+SDS_API_BASE_URL=http://localhost:5003/v1/unit_data
+```
+
+From the home directory, run using
+```python
+ python -m scripts.mock_sds_endpoint
```

Comment:
```suggestion
python -m scripts.mock_sds_endpoint
```
Really minor but theres a space here thats causing the command to be out of alignment when the doc is rendered